### DISCUSSION TITLE
Mini Hud Not Showing on Screen

### DISCUSSION THREAD

**Discussion by Beers927** [Courseplay/Courseplay_FS22#2924] [Q&A]
When I activate the Mini Hud it doesn't show on the screen.  The mouse cursor comes on screen but the hud isnt displayed. What do i need to change to make it visible?

**Comment by Tensuko** [comment]
Your coordinates might be broken.
Check the modSettings folder there is a cp folder where is a settings xml.
Delete that file and start your game.

**Reply by VikPilaf** [reply]
The same thing happens to me too, in the folder you told me there is no file of that type.

**Reply by Tensuko** [reply]
Ok we continue here then.
Attach your logfile pls ( https://github.com/Courseplay/Courseplay_FS22/wiki )

**Reply by Reeses223** [reply]
> Your coordinates might be broken. Check the modSettings folder there is a cp folder where is a settings xml. Delete that file and start your game.

I deleted the "courseplay.xml" file in the "modsettings" folder and then started the game. that worked. thank you!

**Comment by caban58** [comment]
I have the same thing happening now, mouse click on CP bar but it won't activate the minihub, therefore unable to set up fields etc

**Reply by Tensuko** [reply]
So you did read what I wrote abouve and tried it out ?
